<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteList</class>
 <widget class="QWidget" name="NoteList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notes List</string>
  </property>
  <property name="styleSheet">
   <string>QWidget {
    background-color: #f5f7fa;
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, Arial, sans-serif;
    font-size: 14px;
    color: #000000;
}
QLabel {
    color: #000000;
}
QLabel#notesPageTitleLabel {
    font-size: 24px;
    font-weight: 600;
    color: #1a1a1a;
}
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    gridline-color: #e5e7eb;
}
QTableWidget::item {
    padding: 8px;
}
QTableWidget::item:selected {
    background-color: #e5e7eb;
    color: #1f2937;
}
QPushButton {
    background-color: #2563eb;
    color: white;
    border-radius: 6px;
    padding: 12px 24px;
    min-height: 48px;
    font-weight: 500;
    font-size: 15px;
}
QPushButton:hover {
    background-color: #1d4ed8;
}
QPushButton:pressed {
    background-color: #1e40af;
}
QLineEdit {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    padding: 8px 12px;
    min-height: 24px;
}
QLineEdit:focus {
    border: 2px solid #2563eb;
    outline: none;
}
QComboBox {
    background-color: #ffffff;
    border: 1px solid #e2e8f0;
    border-radius: 6px;
    padding: 8px 12px;
    min-width: 200px;
    color: #000000;
}
QComboBox:focus {
    border: 2px solid #2563eb;
    outline: none;
}
QComboBox::drop-down {
    border: none;
    width: 20px;
}
QComboBox::down-arrow {
    image: none;
    border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>32</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>32</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="QLabel" name="notesPageTitleLabel">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <weight>600</weight>
      </font>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>16</number>
      </property>
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchLineEdit">
        <property name="placeholderText">
         <string>Search notes...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeFilterComboBox">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Text</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Doodle</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="notesTableWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date Created</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QPushButton" name="createNoteButton">
       <property name="text">
        <string>Create Note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteNoteButton">
       <property name="text">
        <string>Delete Note</string>
       </property>
       <property name="styleSheet">
        <string>background-color: #e11d48;
color: white;
border-radius: 6px;
padding: 12px 24px;
min-height: 48px;
font-weight: 500;
font-size: 15px;
}
QPushButton:hover {
    background-color: #be123c;
}
QPushButton:pressed {
    background-color: #9f1239;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="backToDashboardButton">
       <property name="text">
        <string>Back to Dashboard</string>
       </property>
       <property name="styleSheet">
        <string>background-color: #e5e7eb;
color: #1a1a1a;
border-radius: 6px;
padding: 12px 24px;
min-height: 48px;
font-weight: 500;
font-size: 15px;
}
QPushButton:hover {
    background-color: #d1d5db;
}
QPushButton:pressed {
    background-color: #9ca3af;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
